lib/index.ts

```diff
@@ -0,0 +1 @@
+export * from './interceptors/json-schema-serializer.interceptor';
```

Comment:
В папке `interceptors` надо создать `index.ts` и там экспортировать все содержимое папки. А здесь надо просто будет экспортировать эту папку

```suggestion
export * from './interceptors';
```

Reply:
+